src/app/modules/activities-management/store/activity-management.reducers.ts

```diff
@@ -56,6 +56,30 @@ export function activityManagementReducer(state: ActivityState = initialState, a
         message: 'Something went wrong creating activities!',
       };
     }
+    case ActivityManagementActionTypes.DELETE_ACTIVITY: {
+      return {
+        ...state,
+        isLoading: true,
+        message: 'Activity removed successfully!',
+      };
+    }
+
+    case ActivityManagementActionTypes.DELETE_ACTIVITY_SUCCESS: {
+      const activites = state.data.filter((activity) => activity.id !== action.activityId);
+      return {
+        data: activites,
+        isLoading: false,
+        message: 'Activity removed successfully!',
+      };
+    }
+
+    case ActivityManagementActionTypes.DELETE_ACTIVITY_FAIL: {
+      return {
+        data: [],
+        isLoading: false,
+        message: 'Something went wrong creating activities!',
```

Comment:
the message is wrong "Something went wrong creating activities!" change for deleting